app/controllers/api/v1/locations_controller.rb

```diff
@@ -10,9 +10,10 @@ def index
 
   def show
     location = Location.find(params["id"])
-    reviews = location.reviews
-
-    payload = {"location":location, "reviews": reviews}
+    all_reviews = location.reviews
+    all_users = Location.findusers(all_reviews)
+    combined_arr = Location.combine(all_reviews, all_users)
+    payload = {"location":location, "reviews": combined_arr}
```

Comment:
Can you clarify why this change was necessary?

Reply:
We needed to pass the information of the user who wrote the review along with the information of the review itself to react. Before the structure of payload wouldn't let us create several ReviewTiles with one map.

Now we combined the info of the review and the user info in one object.